starcounter-include.html

```diff
@@ -247,47 +277,13 @@
          * @param  {Mixed} value New value
          */
         StarcounterIncludePrototype._notifyPath = function(path, value) {
-            var modelAlreadyLoaded = this.template.model == this.partial;
-            if (path.indexOf("partial.") === 0 || path.indexOf("viewModel.") === 0) {
-                if (
-                    // path === "viewModel.Html" || // covered by  viewModel._whatever_
-                    // path.match(/^viewModel\.[^.]+\.Html$/)
-                    (path.indexOf('.Html') === path.length - 5) && path.slice(10, -5).indexOf('.') === -1
-                ) {
-                    this.partialChanged(this.partial);
-                } else if (
-                    // path === "partial.Html" || // covered by  partial._whatever_
-                    // path.match(/^partial\.[^.]+\.Html$/)
-                    (path.indexOf('.Html') === path.length - 5) && path.slice(8, -5).indexOf('.') === -1
-                ) {
-                    this.partialChanged(this.partial);
-                } else {
-                    const compositionProviderPath = findCompositionProviderPath(this.partial);
-                    // if (path === 'partial.' + compositionProviderPath ||
-                    //     path === 'viewModel.' + compositionProviderPath) {
-                    //     // .Composition$ and .PartialId may have changed, also CompositionProvider theoretically could provide it's .Html as well
-                    //     this.partialChanged();
-                    // } else
-                    if(
-                        // .Composition$ have changed
-                        path === 'partial.' + compositionProviderPath  + '.Composition$'||
-                        path === 'viewModel.' + compositionProviderPath + '.Composition$'
-                    ) {
-                        this._compositionChanged();
-                    } else if (path === 'partial.' + compositionProviderPath + '.PartialId' || path === 'viewModel.' + compositionProviderPath + '.PartialId') {
-                        const compositionProvider = findCompositionProvider(this.partial);
-                        this.partialId = compositionProvider && compositionProvider.PartialId; //should always be string
-                    } else {
-                        this.partialChanged(this.partial);
-
-                        if (modelAlreadyLoaded && this.template._notifyPath) {
-                            this.template._notifyPath(
-                                path.replace("partial.", "model.")
-                                    .replace("viewModel.", "model."),
-                                 value);
-                        }
-                    }
-                }
+            var sameModelAlreadyLoaded = this.template.model == this.partial;
+            this.partialChanged(this.partial);
```

Comment:
Should `partialChanged` be called if `sameModelAlreadyLoaded` is true? Maybe slide it inside the following `if` and check `_notifyPath` in another `if`? Esp that `partialChanged` is expensive a bit.

Reply:
> Should `partialChanged` be called if `sameModelAlreadyLoaded` is true?

Yes because it is possible that we have the same model, but one of the `Html` or `Composition$` properties is changed. All the checks in `partialChanged` are needed to check with 100% certainty if HTML of Composition is changed.